client/src/components/screens/PatientSubmission/components/ClinicalInformation/components/MrnItem.tsx

```diff
@@ -26,7 +26,7 @@ interface Props {
 const MrnItem: React.FC<Props> = ({ form, onChange }) => {
   const [mode, setMode] = useState<Mode>(Mode.SELECT);
 
-  const patient = useSelector<State>((state) => state.patient.patient.original) as Patient;
+  const patient = useSelector<State>((state) => state.patientSubmission.patient) as Patient;
```

Comment:
Why the change?

Reply:
When you just created a patient `patient` is empty. And we're in the form so we should use the "right" data